commercial/app/views/books/book.scala.html

```diff
@@ -30,7 +30,7 @@ <h4 class="lineitem__title" itemprop="name">@book.title</h4>
             </p>
             </a>
             <a href="@clickMacro@book.buyUrl" class="lineitem__cta button button--primary button--small" data-link-name="merchandising-bookshop-v@{version}_@{date}-superHigh-visit-shop">
-                Buy now<i class="i i-arrow-white-right i-right"></i>
+                Buy now@fragments.inlineSvg("arrow-white-right", "icon", List("i-right"))
```

Comment:
As discussed, I tried to avoid touching the commercial CSS as much as possible hence adding the `i-right` class to inherit the styles already applied to it for positioning. 

But do agree with the naming and colour convention so will re-name to `arrow-right` now. `arrow` and rotation should be a refactor in another PR.